integration_tests/src/main/python/udf_test.py

```diff
@@ -182,11 +176,37 @@ def group_size_udf(to_process: pd.Series) -> int:
 
 low_upper_win = Window.partitionBy('a').orderBy('b').rowsBetween(-3, 3)
 
-udf_windows = [no_part_win, unbounded_win, cur_follow_win, pre_cur_win, low_upper_win]
+running_win_param = pytest.param(pre_cur_win, marks=pytest.mark.xfail(
+    condition=is_databricks_runtime() and is_spark_341(),
+    reason='DB13.3 wrongly uses RunningWindowFunctionExec to evaluate a PythonUDAF'))
+udf_windows = [no_part_win, unbounded_win, cur_follow_win, running_win_param, low_upper_win]
 window_ids = ['No_Partition', 'Unbounded', 'Unbounded_Following', 'Unbounded_Preceding',
               'Lower_Upper']
 
 
+# This test runs only on CPU to verify that DB13.3 wrongly uses RunningWindowFunctionExec
+# to evaluate a PythonUDAF and always fails due to the following error.
+# """
+#  scala.MatchError: pandas_sum(b#1351)#1354L (of class org.apache.spark.sql.catalyst.expressions.PythonUDAF)
+#      at com.databricks.sql.execution.window.RunningWindowFunctionExec.$anonfun$aggregates$2(RunningWindowFunctionExec.scala:117)
+# """
+# (Not sure if it would be fixed in the future releases.)
+@ignore_order
+@pytest.mark.skipif(not (is_databricks_runtime() and is_spark_341()), reason="only for DB13.3")
+@pytest.mark.parametrize('data_gen', [int_gen], ids=idfn)
+@pytest.mark.parametrize('window', [running_win_param], ids=['Unbounded_Preceding'])
+def test_window_aggregate_udf_on_cpu(data_gen, window):
```

Comment:
Why does this test even exit? Is this supposed to detect when databricks fixes their bug? If so it will just end up with an xpass and we will ignore it. If you want that we need to verify that it failed in the way we expect it to and fail the test if it does not.